site/multiembedder/MultiEmbedder.tsx

```diff
@@ -118,13 +140,18 @@ const figuresFromDOM = (
 export const shouldProgressiveEmbed = () =>
     !isMobile() ||
     window.screen.width > 680 ||
-    pageContainsGlobalEntityControl()
+    pageContainsGlobalEntitySelector()
+
+const pageContainsGlobalEntitySelector = () =>
+    document.querySelector(`[${GLOBAL_ENTITY_SELECTOR_DATA_ATTR}]`) !== null
 
-const pageContainsGlobalEntityControl = () =>
-    document.querySelector(`[${GLOBAL_ENTITY_CONTROL_DATA_ATTR}]`) !== null
+const globalEntitySelectorSelector = () =>
```

Comment:
```suggestion
const globalEntitySelectorElement = () =>
```
may be a better name?

Reply:
ah yes, that was a temp placeholder -- this is better